.github/workflows/msbuild.yml

```diff
@@ -47,9 +47,17 @@ jobs:
         cp -r z3* dafny/Binaries/z3
     - name: Nuget Restore Dafny
       run: nuget restore dafny/Source/Dafny.sln
+    - name: Build Dafny Parser
```

Comment:
Make sure you add this to INSTALL.md and any other place we document building Dafny (the wiki has at least one page too)

Reply:
Added comments, add a separate issue to update the complete INSTALL instructions